tests/input/heic/test_heic.py

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8 -*-
+
+import json
+import os
+import re
+import shutil
+import typing
+
+from PIL import Image
+import pytest
+
+from preview_generator.exception import UnavailablePreviewType
+from preview_generator.manager import PreviewManager
+from preview_generator.utils import imagemagick_supported_mimes
+from tests import test_utils
+
+CURRENT_DIR = os.path.dirname(os.path.abspath(__file__))
+CACHE_DIR = "/tmp/preview-generator-tests/cache"
+IMAGE_FILE_PATH = os.path.join(CURRENT_DIR, "test_heic.heic")
+
+if imagemagick_supported_mimes().count("image/heic") == 0:
+    pytest.skip("heif-convert is not available.", allow_module_level=True)
```

Comment:
This check is nice but this actually mean heic test are currently skipped in travis, so support for heic can be broken silently.

Reply:
Yes, if we want to test HEIC on travis we need to compile our own IM version...

Reply:
Seems supported by default in debian buster: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=914120, hope next ubuntu travis supported version will activated it by default.

Reply:
Using https://github.com/david-poirier-csn/pyheif allow to bypass this issue, but I doesn't work for Python 3.5 so I prefer to wait. We already using this builder without issue with IM ;)